### DISCUSSION TITLE
Add real time capabilities?

### DISCUSSION THREAD

**Discussion by mariolorente** [noam-honig/radweb#66] [General]
It would be awesome if you can subscribe to any model/list. For me it would a no brainer to use this.

**Comment by yoni-rapoport** [comment]
Thanks for posting!

That's on the roadmap and we're getting there soon. Stay tuned.

**Comment by noam-honig** [comment]
Just wanted to share with you that in our last meetup in Budapest I've presented a preview of real-time capabilities we call "live query"

Here's the video:
https://youtu.be/CnCaMQCu3Kc?t=1065

**Comment by yoni-rapoport** [comment]
> It would be awesome if you can subscribe to any model/list. For me it would a no brainer to use this.

Well @mariolorente , it took us a while to make it as great as we wanted, and it's finally out in version [0.18](https://github.com/remult/remult/releases/tag/v.0.18.0). See details in [new tutorial section](https://remult.dev/tutorials/react/live-queries.html).
